domain/network/service/interface.go

```diff
@@ -38,15 +38,21 @@ type State interface {
 type SpaceState interface {
 	// AddSpace creates a space.
 	AddSpace(ctx context.Context, uuid string, name string, providerID network.Id, subnetIDs []string) error
-	// GetSpace returns the space by UUID.
+	// GetSpace returns the space by UUID. If the space is not found, an error
+	// is returned satisfying networkerrors.SpaceNotFound.
```

Comment:
Please replace with `[github.com/juju/juju/domain/network/errors.SpaceNotFound]` and for the ones below.